src/render-delaying-services.js

```diff
@@ -60,10 +89,25 @@ export function waitForServices(win) {
     return Services.timerFor(win).timeoutPromise(
         LOAD_TIMEOUT,
         getServicePromise(win, service),
-        `Render timeout waiting for service ${service} to be ready.`
+        `Render timeout waiting for service ${service} ` +
+      'to be ready.'
     );
   });
-  return Promise.all(promises);
+  return Promise.all(promises).then(services => {
```

Comment:
It looks like there should be a collection of services and only a single timeout promise to force rendering?